packages/middleware-retry/src/config.ts

```diff
@@ -0,0 +1,15 @@
+export enum RETRY_MODES {
+  standard = "standard",
+  adaptive = "adaptive",
```

Comment:
change to CAPS for keys.
```suggestion
  STANDARD = "standard",
  ADAPTIVE = "adaptive",
```

Reply:
Added in [`878c617` (#2454)](https://github.com/aws/aws-sdk-js-v3/pull/2454/commits/878c6178748f66d545e530c517979d98ae69e941)